e2e-tests/cypress/tests/support/okta_commands.ts

```diff
@@ -1,12 +1,47 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {ChainableT} from 'tests/types';
 import * as TIMEOUTS from '../fixtures/timeouts';
 
 const token = 'SSWS ' + Cypress.env('oktaMMAppToken');
 
-function buildProfile(user) {
-    const profile = {
+type UserId = string | null;
+
+interface Profile {
```

Comment:
Profile and User seems to be the same data structure, right? They only change on the whether they have `password` or not. Any strong reason to keep them separated?

Reply:
Even though both interfaces are similar, they differ in the property names in camel-case

`firstName` vs `firstname`